src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -0,0 +1,306 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="[MODFQMMGR-176] Create supporting address categories view" author="novercash@ebsco.com">
+    <preConditions onFail="CONTINUE">
+      <tableExists tableName = "organization_types" schemaName="${tenant_id}_mod_organizations_storage"/>
+    </preConditions>
+    <createView
+      replaceIfExists="true"
+      viewName="src_organization_categories">
+      SELECT * FROM ${tenant_id}_mod_organizations_storage.categories
+    </createView>
+  </changeSet>
+  <changeSet id="[MODFQMMGR-176] Create entity type for organizations contact info" author="novercash@ebsco.com" runOnChange="true">
+    <!-- Insert the row, then fill it in later, for readability. -->
+    <sql>
+      INSERT INTO entity_type_definition (id, derived_table_name, definition)
+      VALUES ('7a7860cd-e939-504f-b51f-ed3e1e6b12b9', 'drv_organization_contacts', '{}') ON CONFLICT (id) DO NOTHING;
+    </sql>
+
+    <update tableName="entity_type_definition">
+      <column name="definition">
+        {
+          "id": "7a7860cd-e939-504f-b51f-ed3e1e6b12b9",
+          "name": "drv_organization_contacts",
+          "private": false,
+          "fromClause": "src_organizations AS org",
+          "columns": [
+            {
+              "name": "id",
+              "dataType": {
+                "dataType": "rangedUUIDType"
+              },
+              "valueGetter": "org.id",
+              "visibleByDefault": true
+            },
+            {
+              "name": "last_updated",
+              "dataType": {
+                "dataType": "dateType"
+              },
+              "valueGetter": "org.jsonb->'metadata'->>'updatedDate'",
+              "visibleByDefault": false
+            },
+            {
+              "name": "name",
+              "dataType": {
+                "dataType": "stringType"
+              },
+              "filterValueGetter": "${tenant_id}_mod_organizations_storage.f_unaccent(org.jsonb ->> 'name'::text)",
```

Comment:
One thing that concerns me about this is that we've been moving towards handling case sensitivity with filterValueGetters and valueFunctions. This was working fine when the indexes we used for filterValueGetters were converting everything to lowercase, but it seems like that's not the case here. So I'm wondering if we're going to lose case insensitivity in this situation.

Not that there's anything that you can do about it in this ticket, but something we might want to be aware of moving forward.